frontend/views/pages/GroupSettings.vue

```diff
@@ -146,8 +159,17 @@ export default ({
         contractID: this.$store.state.currentGroupId,
         key: 'groupPicture'
       }
+    },
+    isGroupAdmin () {
+      // TODO: group admin doesn't mean group creator
+      // https://github.com/okTurtles/group-income/issues/202
+      return this.groupSettings.groupCreator === this.ourUsername
     }
   },
+  mounted () {
+    // TODO: need to remove double exclamation mark after release version 1
```

Comment:
This comment is unclear. What needs to be check?

Reply:
Wanted to say that `Double exclamation mark should be removed when we release this version and no more previous versions are using. That's because groupSettings.allowPublicChannels could be undefined.`

Reply:
Can I remove this comment and also two exclamation mark too?

Reply:
Sure